drivers/sensor/max17055/max17055.h

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_DRIVERS_SENSOR_BATTERY_MAX17055_H_
+#define ZEPHYR_DRIVERS_SENSOR_BATTERY_MAX17055_H_
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(max17055, CONFIG_SENSOR_LOG_LEVEL);
```

Comment:
Highly unlikely that this file would ever be used by multiple compilation units within the same build (which would cause a conflict), but its still better if the module registration with logging is done in the *.c file..

Reply:
done